docs/cli/secretcli.md

```diff
@@ -230,46 +230,46 @@ Optionally, you can supply your address within the command as:
 secretcli q account $(secretcli keys show -a <key-alias>)
 ```
 
-::: warning Note
```

Comment:
Is there a reason why this is lowercase? @dylanschultzie

Reply:
I answered my own question. Plz ignore the changes to the case for all warning --> Warning changes.